bundles/org.openhab.binding.systeminfo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -295,6 +296,14 @@
 		<config-description-ref uri="channel-type:systeminfo:highpriority_process"/>
 	</channel-type>
 
+	<channel-type id="cpuLoad">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
You are setting `Number:Dimensionless` but you are building a DecimalType state to update the channel state.
So `Number` looks sufficient.